### PR TITLE
add external app hosting guide TECHCOMM-167

### PR DESCRIPTION
**Pull request by anslzo**
This uses MrMurano as the CLI tool, so maybe this should wait to be integrated until something happens with MrMurano.

### CODE REVIEW

**Comment by anslzo** [comment]
@chadandycha - can you review this?

File: guides/external-app-hosting/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+template: default
+---
+
+# Guide: Using Murano with your mobile app or existing web frontend
+
+Murano can be used with mobile and existing frontend applications provided the CORS settings on your solution are configured properly.
+
+This guide will go through configuring your CORS file with the `murano` CLI tool.
+
+Cross Origin Request Sharing (CORS) is a way to mitigate cross origin attacks in the browser. MDN has a good resource explaining [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Access_control_CORS).
```
Comment (endthestart) [inline guides/external-app-hosting/README.md]: Mozilla Developer Network (MDN) has a good...

File: guides/external-app-hosting/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+### CORS by Example
+
+In this example, we will make requests from `https://exosite.com` to our solution API which is hosted at `https://cors.apps.exosite.io`. We will only be using the API portion of the solution, it will not be hosting any frontend files. Depending on the initial configuration of CORS in your solution, these requests may succeed (if `origin: true` is set in your `cors.yaml` file).
+
+#### Initial assumptions
+- We have a solution.
```
Comment (endthestart) [inline guides/external-app-hosting/README.md]: Provide links to these guides.

File: guides/external-app-hosting/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+$ cat endpoints/test.lua
+--#ENDPOINT GET /test
+return 'success!'
+~~~
+
+**Ensure `origin: true` is not present in the `cors.yaml` file.** If `origin` is set to `true`, all requests from all domains will be allowed. If your default cors.yaml file is configured this way, modify the origin line to the following:
+~~~
+origin:
+- http://localhost:3000
+- https://cors.apps.exosite.io
```
Comment (endthestart) [inline guides/external-app-hosting/README.md]: Should be exosite.com

**Pull request merged**